build/storage/tests/it/ptf_tests/system_tools/config.py

```diff
@@ -57,4 +64,6 @@ def __init__(self):
             self.password = os.getenv("HOST_TARGET_PASSWORD")
```

Comment:
you are using the same initialization 4 times - looks like a candidate for abstraction?

Reply:
changed